src/lifecycle/properties-ready.js

```diff
@@ -1,6 +1,5 @@
 export default function propertiesApply (elem, properties) {
   Object.keys(properties).forEach(function (name) {
-    let prop = properties[name];
-    prop.set && prop.set.call(elem, elem[name]);
+    properties[name].init.call(elem);
```

Comment:
Does this mean that `set` is no longer called when the element is initialised? Does this have any other side effects? (It looks like `init` doesn't do a lot of things that `set` did)